src/main/java/com/google/api/codegen/discovery/transformer/php/PhpSampleMethodToViewTransformer.java

```diff
@@ -169,7 +170,14 @@ private SamplePageStreamingView createSamplePageStreamingView(
     builder.isResourceMap(fieldInfo.type().isMap());
     builder.pageVarName(
         symbolTable.getNewSymbol(namer.localVarName(Name.lowerCamel(fieldInfo.name()))));
-    builder.pageTokenName(methodInfo.requestPageTokenName());
+    boolean isResourceSetterInRequestBody = methodInfo.isPageStreamingResourceSetterInRequestBody();
+    builder.isResourceSetterInRequestBody(isResourceSetterInRequestBody);
+    // Use upper camel case for getter/setter function names in templates.
+    if (isResourceSetterInRequestBody) {
+      builder.pageTokenName(Name.lowerCamel(methodInfo.requestPageTokenName()).toUpperCamel());
```

Comment:
Oh, I see, that's how I did it in Java, huh? :/

If you could make an issue then so I can clean it up later that'll be enough.